lib/Agrammon/Web/Routes.pm6

```diff
@@ -282,9 +282,12 @@ sub application-routes(Agrammon::Web::Service $ws) {
 
         # test
         post -> LoggedIn $user, 'store_variable_comment' {
-            request-body -> (:$dataset, :$variable, :$comment) {
-                my $data = $ws.store-input-comment($user, $dataset, $variable, $comment);
-                content 'application/json', $data;
+            request-body -> %data {
+                my $ret = $ws.store-input-comment(
+                    $user,
+                    :dataset(%data<dataset>), :variable(%data<variable>), :comment(%data<comment>)
```

Comment:
Probably also better unpacked in the `request-body` line.

Reply:
Done